setup.py

```diff
@@ -213,6 +213,10 @@ def cuda_ext_helper(name, sources, extra_cuda_flags, extra_cxx_flags=[]):
     ext_modules=ext_modules,
     cmdclass={'build_ext': BuildExtension} if ext_modules else {},
     install_requires=fetch_requirements('requirements/requirements.txt'),
+    entry_points='''
+        [console_scripts]
+        colossal=colossalai.scripts.cli:cli
```

Comment:
Is there any purpose of creating two-level directories? I think the norm is to have a folder directly under the main repo, e.g. `/colossalai/cli`